packages/flutter/lib/src/widgets/unique_widget.dart

```diff
@@ -19,7 +19,9 @@ abstract class UniqueWidget<T extends State> extends StatefulWidget {
   }
 
   @override
-  T createState();
+  T createState(); //ignore: strong_mode_invalid_method_override
+  // TODO(abarth): Having to ignore strong_mode_invalid_method_override here
+  // seems like a bug. We should remove the ignore once we sort out the issue.
```

Comment:
try the thing i sent by e-mail, but if that doesn't work, file a bug and include the bug# here